.ncurc.json

```diff
@@ -1,6 +1,4 @@
 {
-    "upgrade": true,
-    "reject": [
-      "@types/vscode"
-    ]
+  "upgrade": true,
+  "reject": ["@types/vscode"]
```

Comment:
Any way not to inline lists?

Reply:
No idea as I did not look into it. I see no problem having short lists inline and I am sure that once it will grow enough it will reformat it on multiple lines.

Reply:
As I said above, lists are naturally multiline things, it's not like we need a compressed JSON. It's not resolved. Don't mark it as resolved arbitrarily. I think it's important to consistently have lists show up as lists.